doc/rippled-example.cfg

```diff
@@ -273,6 +273,31 @@
 #               keep rippled from connecting to other instances of rippled or
 #               prevent RPC and WebSocket clients from connecting.
 #
+# WebSocket permessage-deflate extension options
+#
+#   These settings configure the optional permessage-deflate extension
+#   options and may appear on any port configuration entry. They are meaningful
+#   only to ports which have enabled a WebSocket protocol.
+#
+#   permessage_deflate = <flag>
+#
+#       Determines if permessage_deflate extension negotiations are enabled.
+#       When enabled, clients may request the extension and the server will
+#       offer the enabled extension in response.
+#
+#   client_max_window_bits = [9..15]
+#   server_max_window_bits = [9..15]
+#   client_no_context_takeover = <flag>
+#   server_no_context_takeover = <flag>
+#   compress_level = [1..9]
+#   memory_level = [1..9]
+#
+#       These optional settings control options related to the
+#       permessage-deflate extension negotiation. For precise
+#       definitions of these fields please see the RFC 7692,
+#       "Compression Extensions for WebSocket":
+#           https://tools.ietf.org/html/rfc7692
```

Comment:
Whoops, yeah - that's a property of the implementation and not the specification.
Willfix